### PR TITLE
Retrieve caffe version from caffe.exe.

### PR DESCRIPTION
**Pull request by IsaacYangSLA**
Try to retrieve caffe version from command line option '-version' and if caffe.exe is not built with such information, fallback to default (0,11,0).

### CODE REVIEW

File: digits/templates/layout.html [hunk 1]
```diff
@@ -51,7 +51,8 @@
                 {% if server_name %}
                 <li><span class="navbar-text">Server: {{server_name}}</span></li>
                 {% endif %}
-                <li><span class="navbar-text">Version: {{server_version}}</span></li>
+                <li><span class="navbar-text">Server ver.: {{server_version}}</span></li>
+				<li><span class="navbar-text">Caffe ver.: {{caffe_version}}</span><li>
```
Comment (lukeyeager) [inline digits/templates/layout.html]: Spacing looks weird here

File: digits/config/caffe_option.py [hunk 2]
```diff
@@ -192,8 +192,25 @@ def get_version(executable):
...
+                raise config_option.BadValue(p.stderr.read().strip())
+            else:
+                pattern = 'version'
+                parsed_version = parse_version(0,11,0)
+                for line in p.stdout:
+                    if pattern in line:
+                        try:
+                            # if CAFFE for Windows is built with properly formatted version string
+                            parsed_version = parse_version(line[line.find(pattern)+len(pattern):].rstrip())
+                        except:
```
Comment (lukeyeager) [inline digits/config/caffe_option.py]: Please don't ever use a bare `except`/`pass`. If you're expecting a particular type of error, catch it like:
```py
except ValueError:
  pass
```

File: digits/templates/layout.html (hunk 1 above)
Comment (lukeyeager) [inline digits/templates/layout.html]: ^ I meant the whitespace in the code.

But also, it looks kinda cramped and unreadable right now. How about this instead?

caffe-version-ui

**Comment by IsaacYangSLA** [comment]
The suggested Info menu layout is much better.  I changed that file accordingly.  Like the following screen shot.
image

**Pull request merged**